glide.yaml

```diff
@@ -12,3 +12,5 @@ import:
   - go/ast/astutil
 - package: github.com/jessevdk/go-flags
 - package: github.com/anmitsu/go-shlex
+- package: go.uber.org/multierr
+  version: ~0.2.0
```

Comment:
Not necessarily. This is never exposed directly to users. As long as we're pinning to it, we're fine. Worst case, we actually check in the vendored code.